website/docs/topics/feature-flags/best-practices-using-feature-flags-at-scale.md

```diff
@@ -0,0 +1,350 @@
+
+## Common pitfalls to avoid when implementing flags at scale
+
+This section could have been called “Been There. Done That. Got the T-shirt.” We’ve seen it all when it comes to large-scale flag implementations, and here are some of the things that seem like a good idea at the time but come back to bite you at scale.
+
+### 11. Avoid feature flag parent-child dependencies.
+
+For larger organizations, it is often tempting to group multiple feature flags together and establish a “parent-child” dependency. The purpose of this relationship is to make it easier to coordinate releases. Often, you would like to have a single parent that will enable/disable multiple features that work together in concert. 
+
+Dependent feature flags make sense when you have different parts of a complex feature you want the option to disable individually but want the ability to control the gradual rollout strategies for each part of the feature together. The parent flag also acts as a global kill switch for the sibling feature flags.
+
+However, you should be careful when introducing these dependencies, as it increases the complexity of everyone interacting with the feature flag system. It's no longer enough to consider the configuration of a single flag, you also need to consider the configuration of the parent. It's easy to cause confusion, leading to accidentally enabling or disabling a feature in production 
+
+To keep complexity down, we generally advise not to have complex targeting rules for both the parent and the child flag simultaneously. This can make it harder to achieve consistency for end users. It also increases the complexity for the user of the feature flag system. For example, if both the parent and the child are configured to a 50% rollout, are you then exposing the feature to 50% or 25% of the user base? The answer is that it depends on the identifier used to roll the flag out.
+
+### 12. Don’t use flags to manage configuration.
+
+Both feature flags and configuration settings control an application’s behavior. However, it's crucial to distinguish between feature flagging systems and configuration management systems and use each for its intended purpose.  Otherwise, you risk management nightmares and security vulnerabilities
+
+Feature flags are intended for dynamic and temporary use, giving you [runtime control](https://docs.getunleash.io/topics/feature-flags/runtime-control) to enable or disable functionality in a live environment. This flexibility supports [short-lived experiments](https://docs.getunleash.io/topics/feature-flags/short-lived-feature-flags), such as A/B testing or gradual rollouts, instead of relying on configuration settings that require an application redeploy to take effect. Despite the ease of turning off and on features or functionality with feature flags, they should not be repurposed for managing configuration.
```

Comment:
the links here also need to be updated (as a separate point to Alvin's):

```suggestion
Feature flags are intended for dynamic and temporary use, giving you [runtime control](feature-flag-best-practices#1-enable-run-time-control-control-flags-dynamically-not-using-config-files) to enable or disable functionality in a live environment. This flexibility supports [short-lived experiments](feature-flag-best-practices#7-make-feature-flags-short-lived-do-not-confuse-flags-with-application-configuration), such as A/B testing or gradual rollouts, instead of relying on configuration settings that require an application redeploy to take effect. Despite the ease of turning off and on features or functionality with feature flags, they should not be repurposed for managing configuration.
```